The references are from the runtime:

> Undefined symbols for architecture x86_64:
>   "__T0s13_getErrorCodeSiSPyxGs0B0RzlF", referenced from:
>       __swift_stdlib_bridgeErrorToNSError in ErrorObject.mm.o
>   "__T0s23_getErrorDomainNSStringyXlSPyxGs0B0RzlF", referenced from:
>       __swift_stdlib_bridgeErrorToNSError in ErrorObject.mm.o
>   "__T0s29_getErrorUserInfoNSDictionaryyXlSgSPyxGs0B0RzlF", referenced from:
>       __swift_stdlib_bridgeErrorToNSError in ErrorObject.mm.o
>   "__T0s32_getErrorEmbeddedNSErrorIndirectyXlSgSPyxGs0B0RzlF", referenced from:
>       dynamicCastValueToNSError(swift::OpaqueValue*, swift::TargetMetadata<swift::InProcess> const*, swift::WitnessTable const*, swift::DynamicCastFlags) in Casting.cpp.o
>       _dynamicCastToExistential(swift::OpaqueValue*, swift::OpaqueValue*, swift::TargetMetadata<swift::InProcess> const*, swift::TargetExistentialTypeMetadata<swift::InProcess> const*, swift::DynamicCastF

These show up when building:
unittests/runtime/LongTests/SwiftRuntimeLongTests
unittests/runtime/SwiftRuntimeTests

--- a/unittests/runtime/Stdlib.cpp
+++ b/unittests/runtime/Stdlib.cpp
@@ -119,3 +119,23 @@ bool _swift_dictionaryDownCastConditionalIndirect(OpaqueValue *destination,
                                         const void *targetKeyHashable) {
   abort();
 }
+
+SWIFT_CC(swift) SWIFT_RUNTIME_STDLIB_INTERNAL
+int _T0s13_getErrorCodeSiSPyxGs0B0RzlF(void *) {
+  abort();
+}
+
+SWIFT_CC(swift) SWIFT_RUNTIME_STDLIB_INTERNAL
+void *_T0s23_getErrorDomainNSStringyXlSPyxGs0B0RzlF(void *) {
+  abort();
+}
+
+SWIFT_CC(swift) SWIFT_RUNTIME_STDLIB_INTERNAL
+void *_T0s29_getErrorUserInfoNSDictionaryyXlSgSPyxGs0B0RzlF(void *) {
+  abort();
+}
+
+SWIFT_CC(swift) SWIFT_RUNTIME_STDLIB_INTERNAL
+void *_T0s32_getErrorEmbeddedNSErrorIndirectyXlSgSPyxGs0B0RzlF(void *) {
+  abort();
+}

